kernel/setup/ezsetupcommon.php

```diff
@@ -29,7 +29,13 @@ function eZSetupOptionalTests()
 
 function eZSetupDatabaseMap()
 {
-    return array( 'mysqli' => array( 'type' => 'mysqli',
+    return array( 'sqlite' => array( 'type' => 'sqlite',
```

Comment:
Is that affecting the order in the frontend? Is 'sqlite' the first choice? I think we want to have it as the 3rd choice.

Reply:
Addressed by moving sqlite down in order.